### DISCUSSION TITLE
Soroban CLI basics: deployment, playing with more complex invocations, and signatures

### DISCUSSION THREAD

**Discussion by heytdep** [stellar/sorobanathon#38] [General]
Hey, we wrote a write-up about writing a contract, deploying it to futurenet/local network, and interacting with it through the soroban CLI. Readers will learn how to use the CLI to complete tasks like deploying, simple invocations, custom structures and simple authentication. 

The write-up doesn't go in-depth on building signatures by signing a payload with ed25519 keys for simplicity since it currently has to be done by generating signatures from a rust test and then passing the object to the CLI.

The write-up is in the readme of [this repo](https://github.com/Xycloo/soroban-cli-futurenet), or you can also read it at https://heytdep.github.io/comp_posts/9)--pub)--Soroban-CLI-basics:-deployment-and-playing-with-invocations/post.html.

**Comment by achallagundla** [comment]
As always super grateful for these awesome submissions!! Will get back to you soon on your reward! :)

**Comment by achallagundla** [comment]
Heya @heytdep ,
Congratulations! Your contribution to Sorobanathon: First Light has been accepted! 🎉
Your submission may qualify for a reward of 1100 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions. (You can also message me on discord @ anuxhya)

All the best,
Anuhya

PS: No need to submit the form if you already have, just update me on discord if your wallet address has changed.
